.graphqlrc.yml

```diff
@@ -0,0 +1,12 @@
+schema: https://raw.githubusercontent.com/rkpattnaik780/enmasse/14c4e82a0c13370d5e14a74669982e1a1105a0ba/console/console-init/ui/mock-console-server/schema.graphql
```

Comment:
Do we have actual schema available somewhere? This looks like enmasse.

Reply:
Agreed - worth getting one into https://github.com/strimzi/strimzi-admin as a part of this issue?

Reply:
This schema can then be used/referenced in our MockApi too (in effect completing #50 )

Reply:
> Do we have actual schema available somewhere? This looks like enmasse.

yes, This is `enmasse` schema. I suggested it for testing purpose. Later we can point out the actual schema.

Reply:
@matthew-chirgwin Do you have some barebones of the schema. Can I take look on it. 
Little bit context. I'm involved in GraphQL spec and working on couple other standards like https://graphql-crud.org 

We working on internal company standards for GraphQL schema and being involved in the process for creating schema for strimzi UI will be helpful..

Reply:
Sorry for the delay in replying @wtrocki - we have a very barebones/placeholder schema here: https://github.com/strimzi/strimzi-ui/blob/e4499460c95472438309a6544084879e079ce27e/server/mockapi/data.ts#L7 , but this will need to be refined in collaberation with @gmcrobert , who will be implementing the 'real' version of it

Reply:
@wtrocki I am looking at defining the schema and would be interested in chatting to you about standards around things like identity, paging, etc. Shall I start a conversation in the strimzi-ui-dev slack channel?

Reply:
Yes. Feel free to ping me anywhere.
As for standards we have been using (and promoting) https://graphqlcrud.org

This comes with generators to reduce boilerplate etc. I have been demoing this already using

https://github.com/wtrocki/strimzi-experiments

See model.graphql as input and finalSchema which is the output. There is numerous benefits of using that - automatic mocking capability that works with postgres and mongo etc.